lib/lib-search/src/main/java/org/sagebionetworks/search/SearchDaoImpl.java

```diff
@@ -130,18 +124,26 @@ public void createOrUpdateSearchDocument(List<Document> batch) throws ClientProt
 			for(Document document: documents){
 				prepareDocument(document);
 				serializedDocument = EntityFactory.createJSONStringForEntity(document);
-				// AwesomeSearch pukes on control characters. Some descriptions have
-				// control characters in them for some reason, in any case, just get rid
-				// of all control characters in the search document
-				String cleanedDocument = serializedDocument.replaceAll("\\p{Cc}", "");

```

Comment:
I agree with this assessment after looking upstream, but I'll bounce this off of John. Thanks!